README.md

```diff
@@ -18,7 +17,25 @@
         def application do
           [applications: [:phoenix_pubsub]]
         end
-
+        
+        
+## Initialisation (without Phoenix)
+
+
+        defmodule MyApp do
+          use Application
+        
+          def start(_type, _args) do
+            import Supervisor.Spec, warn: false
+        
+            children = [
+              supervisor(Phoenix.PubSub.PG2, [MyApp.PubSub, [pool_size: 1]])
```

Comment:
Perhaps we should leave out the `pool_size` option? It defaults to the number of schedulers now.